api/src/main/java/com/nicknathanjustin/streamercontracts/settings/UserSettingsDto.java

```diff
@@ -0,0 +1,13 @@
+package com.nicknathanjustin.streamercontracts.settings;
+
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
```

Comment:
I know i was the bad boy that introduced @Data on the DTOs... @Value is technically a little better for us. Consider changing. (Im making sure to flip all @Data's to @Values when I touch the existing code). 

The big difference between @Data and @Value is @value is meant to be immutable which is how must/all of these DTOs should be